public/stylesheets/cards.css

```diff
@@ -1,20 +1,71 @@
-.image-box {
-  width: 50vw;
+.card {
+  display: flex;
+  justify-content: center;
+  padding: 80px 10px;
+}
+.card-image {
+  background-color: black;
```

Comment:
I'm not sure I understand but what does this attribute do to? Is it needed even?

Reply:
Basically wanted this to be there in cases when the image box is bigger than the image itself (could happen due to image ration), though the way the image is being displayed right not it does nothing. Some changes will happen in regards to this at a later point.